model-optimizer/mo/utils/cli_parser.py

```diff
@@ -254,6 +253,14 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'and biases are quantized to FP16.',
                               choices=["FP16", "FP32", "half", "float"],
                               default='float')
+    common_group.add_argument('--transform',
+                              help='Apply additional transformations. ' +
+                                   'Usage: "--transform pass_name1[args],pass_name2..." ' +
```

Comment:
may be change "pass_name1" to "trans_name1"? Because the word "pass" looks ambiguous in this context.

Reply:
Let's use `transformation_name`. Anyway it will look like a mess:
```
Before:
  --transform TRANSFORM
                        Apply additional transformations. Usage: "--transform
                        pass_name1[args],pass_name2..." where [args] is
                        key=value pairs separated by semicolonExamples: "--
                        transform LowLatency" or "--transform
                        LowLatency[num_iterations=2]" Available
                        transformations: "LowLatency"

After:
  --transform TRANSFORM
                        Apply additional transformations. Usage: "--transform
                        transformation_name1[args],transformation_name2..."
                        where [args] is key=value pairs separated by
                        semicolon. Examples: "--transform LowLatency" or "--
                        transform LowLatency[num_iterations=2]" Available
                        transformations: "LowLatency"
```